### PR TITLE
Completed weed two assessment

### PR DESCRIPTION
**Pull request by R1que007**
# Submitting Week 2 Assessment

### ✅ Completed Files

Select the following as confirmation that the files in this repository are complete.

- [✅ ] Career Readiness
- [✅ ] Coding Challenges
- [✅ ] Coding Critique
- [✅ ] Coding Concepts
- [✅ ] Interview Questions

### ✋ Reminder

Do NOT merge your assessment!

### 📝 Feedback

If you would like specific feedback please leave details here:

### CODE REVIEW

File: code-challenges.test.js [hunk 1]
```diff
@@ -9,43 +9,105 @@
...
+    it("decides if a number is evenly divisible by three", () => {
+    const object1 = { number: 15 }
+    //Expected output: "true"
+    const object2 = { number: 0 }
+    //Expected output: "false"
+    const object3 = { number: -7 }
+    //Expected output: "false"
+    expect(divBy3(object1)).toEqual(true)
+    expect(divBy3(object2)).toEqual(true)
+    expect(divBy3(object3)).toEqual(false)
```
Comment (chrisyhlee) [inline code-challenges.test.js:58]: Pay attention to what the prompt says the function should return. This test says the function is returning a Boolean, but the prompt says it should return a string. Don't just change the expected output to fit your code.

File: code-critique.js [hunk 2]
```diff
@@ -6,5 +6,23 @@
...
+// Things to consider in your critique: Were you surprised by the output? Was the output what you expected? Is the correct output being rendered? What do you like about the code? What do you dislike? Are there best practices that are not being followed? Would you take a different approach? What did you learn? 
+
+//First, I noticed that the code is not javascript. I didn't think about what possible output I might get. I did not expect the output to be written in javascript because the prompt did not specify any coding language. I am suprised that I can understand this and follow the code. The code defines what a vowel is. It checks the value at i and if there is a vowel, it returns that value, otherwise , return is -1 if not a vowel. I am guessing "print" is similar to "console.log()". If a vowel is present, the code uses string interpolation to add the index to the string/message it returns. If no vowel is present, it retrurns only a string, "No vowels found in the string."
+// I learned that when using AI, we must be very specific. It is only a tool to assist us in research and to assist us in writing code. It is a vluable tool if we use it with caution.
```
Comment (chrisyhlee) [inline code-critique.js:28]: Great takeaway.

**Review by chrisyhlee — APPROVED** [review]
Make sure to pay attention to details! For challenge 1, the output should be a string, not a Boolean. I left a comment on that as well. Other than that, assessment looks good! This currently displays Novice. The part holding you back the most is the interview questions, make sure you are elaborating on your answers, as well as the researched answers. You don't want to go to an interview and make the interviewer have doubts that you know what you are talking about. Keep up the great work.

**Comment by R1que007** [comment]
Thanks for the feedback.

**Pull request merged**